src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/validator/CreateCertifiedCopyItemRequestValidator.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.validator;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.certifiedcopies.orders.api.dto.CertifiedCopyItemOptionsRequestDTO;
+import uk.gov.companieshouse.certifiedcopies.orders.api.dto.CertifiedCopyItemRequestDTO;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.CertifiedCopyItem;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.CertifiedCopyItemOptions;
+import uk.gov.companieshouse.certifiedcopies.orders.api.util.FieldNameConverter;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Sonar report indicates unused imports. Can the unused imports be removed please?

Reply:
These have been removed.